src/ui/components/Carousel/index.js

```diff
@@ -0,0 +1,47 @@
+/* @flow */
+import NukaCarousel from 'nuka-carousel';
+import React from 'react';
+import { compose } from 'redux';
+
+import translate from 'core/i18n/translate';
+import { getDirection } from 'core/i18n/utils';
+import Card from 'ui/components/Card';
+
+import './styles.scss';
+
+
+type PropTypes = {|
+  i18n: Object,
+  // eslint-disable-next-line no-undef
+  sections: Array<React$Element<*>>,
```

Comment:
So I tested it locally and I think we should be using:

```
  sections: Array<React.Element<typeof CarouselSection>>,
```

Sorry this changes often...

Reply:
No worries, I'll try that.